room_assistance/indico_room_assistance/plugin.py

```diff
@@ -91,3 +96,27 @@ def _extend_services_menu(self, reservation, **kwargs):
 
     def _get_room_assistance_request(self, sender, **kwargs):
         return RoomAssistanceRequest
+
+    def _on_event_update(self, event, **kwargs):
+        changes = kwargs['changes']
+        if not changes.viewkeys() & {'location_data', 'start_dt', 'end_dt'}:
+            return
+
+        request = Request.find_latest_for_event(event, RoomAssistanceRequest.name)
+        if not request or request.state != RequestState.accepted:
+            return
+
+        if 'location_data' in changes and not event_has_room_with_support_attached(event):
+            flash(_("The new event location is not in the list of the rooms supported by the room assistance team. "
```

Comment:
shouldn't we auto-reject the request in this case?

Reply:
We don't reject it for example in case of the `vc-assistance` request so I thought that it should be the same.

Reply:
hmm i checked in audiovisual and there we do.. maybe we should reject in both cases? if you end up in a state where you wouldn't be able to send a request i think it makes sense if the event managers are clearly notified (the rejection triggers an email to all managers)

Reply:
Ok, then maybe I will do the same in room assistance.